src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c

```diff
@@ -84,6 +86,13 @@ struct ospi_versal {
 #define	OV_INFO(ov, fmt, arg...)    \
 	xocl_info(&ov->ov_pdev->dev, fmt "\n", ##arg)
 
+#ifdef OSPI_VERBOSE
```

Comment:
any concern we need to use #ifdef here? Though without this, we can turn the debug information without recompiling the code.

Reply:
Agree, so just remove the ifdef for deb logs.